www/templates/intro/consent.html

```diff
@@ -1,111 +1,174 @@
-
-<h3>Studies lead by other researchers</h3>
-If this platform is being used to collect data for a study conducted by another
-researcher, for example, from a Transportation Engineering Department, then you
-will be asked to assent to a separate document outlining the data association,
-retention and sharing policies for that study, <b>in addition to the policies
-above</b>. We will make all data, including the mapping between the email
-address and the UUID, directly available to the lead researcher for the main
-study. This will allow them to associate the automatically gathered information
-with demographic data, and any pre and post surveys that they conduct as part
-of their study. The other researcher may also choose to compensate you for your
-time, as described in the protocol document for that study.
-
-<h3>Your rights</h3>
-<b><i>Participation in research is completely voluntary</i></b>.  You have the
-right to decline to participate or to withdraw at any point in this study
-without penalty or loss of benefits to which you are otherwise entitled.
-
-<h3>Questions</h3>
-If you have any questions about this research, please feel free to contact us.
-For the quickest response, you can send email to the entire team at <a
-href="mailto:e-mission@lists.eecs.berkeley.edu">e-mission@lists.eecs.berkeley.edu</a>.
-If you want to contact only the lead researcher or the PI, you can use their
-profiles above.
-
-If you have any questions about your rights or treatment as a research
-participant using this platform, please contact the University of California at
-Berkeley's Committee for Protection of Human Subjects at 510-642-7461, or
-e-mail <a href="mailto:subjects@berkeley.edu">subjects@berkeley.edu</a>.
+  OpenToAll ("us", "we", or "our") operates the <a
+  href="http://www.accessmap.io">http://www.accessmap.io</a>
+  website (the "Service").<br/>
+  <br/>
+  This page informs you of our policies regarding the collection, use and
+  disclosure of Personal Information when you use our Service.<br/>
+  <br/>
+  We will not use or share your information with anyone except as described in
+  this Privacy Policy.<br/>
+  <br/>
+  We use your Personal Information for providing and improving the Service. By
+  using the Service, you agree to the collection and use of information in
+  accordance with this policy. Unless otherwise defined in this Privacy Policy,
+  terms used in this Privacy Policy have the same meanings as in our Terms and
+  Conditions, accessible at <a
+  href="http://www.accessmap.io">http://www.accessmap.io</a>
+  <br/>
+
+<h3>Information Collection And Use</h3>
+  While using our Service, we may ask you to provide us with certain personally
+  identifiable information that can be used to contact or identify you.
+  Personally identifiable information may include, but is not limited to, your
+  email address, name, phone number, postal address, other information
```

Comment:
Shouldn't you add something about collecting location information in the background? Or is it covered by the "not limited to" clause?

Reply:
Got it. Will state it explicitly.